Ziti.NET.Standard/src/OpenZiti/ZitiIdentity.cs

```diff
@@ -424,6 +436,18 @@ public void ZitiResponse(object evt)
 			await streamLock.WaitAsync().ConfigureAwait(false);
 		}
 
+		public void UpdateControllerUrlInConfigFile(string controller_url) {

```

Comment:
this needs to be reverted. we shouldn't do this on behalf of the developer. just make an event and trigger it with the payload like:
```
    public delegate void OnZitiClientConnected(ZitiConnection serverConnection, ZitiConnection clientConnection, ZitiStatus status);
```